english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md

```diff
@@ -0,0 +1,8 @@
+You suddenly travel to Berlin 1989, lying on the road, at night (without wearing cloth!!).
```

Comment:
Casey is not saying that this file needs a link in it. Rather, the rest of the story needs a link that leads to this file. If a reader starts by choosing their language in `language.md`, then keeps clicking links, you want it to be possible for them to reach this file.